llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -27328,6 +27330,36 @@ void AArch64TargetLowering::ReplaceExtractSubVectorResults(
   Results.push_back(DAG.getNode(ISD::TRUNCATE, DL, VT, Half));
 }
 
+void AArch64TargetLowering::ReplaceGetActiveLaneMaskResults(
+    SDNode *N, SmallVectorImpl<SDValue> &Results, SelectionDAG &DAG) const {
+  if (!Subtarget->hasSVE2p1())
```

Comment:
This should protect the `setOperationAction()` call and then be an assert here, that way the function is only called when the necessary instructions are available.

Do you mind also extending the PR to cover `Subtarget.hasSME2() && Subtarget.isStreaming()`?